<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>702</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>473</width>
    <height>702</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>473</width>
    <height>702</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SelTRon Not Defterim</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>72</y>
      <width>441</width>
      <height>581</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chakra Petch</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
border-radius:5px;
color: rgb(211, 215, 207);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>54</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>******************************************************************************************************</string>
    </property>
   </widget>
   <widget class="QLabel" name="not_basligi">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>23</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chakra Petch</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Başlık Yazısı</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>473</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 159, 207);
border-radius:25px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="secret_button2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>27</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Ekranı Gizle</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:5px;
background-color: rgb(245, 121, 0);
}
QPushButton:hover{
background-color: rgb(245, 121, 160);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="close_button2">
    <property name="geometry">
     <rect>
      <x>432</x>
      <y>27</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Çıkış</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:5px;
background-color: rgb(164, 0, 0);
}
QPushButton:hover{
background-color: rgb(164, 50, 150);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>label</zorder>
   <zorder>not_basligi</zorder>
   <zorder>secret_button2</zorder>
   <zorder>close_button2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
